hack/deploy.sh

```diff
@@ -5,71 +5,46 @@ set -e
 # expect oc to be in PATH by default
 OC_TOOL="${OC_TOOL:-oc}"
 
-# Override the image name when this is invoked from openshift ci                               
-if [ -n "${OPENSHIFT_BUILD_NAMESPACE}" ]; then                                                   
-        FULL_REGISTRY_IMAGE="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:performance-addon-operators-registry"
-        echo "Openshift CI detected, deploying using image $FULL_REGISTRY_IMAGE"                   
-fi   
+# expect kustomize to be in PATH by default
+KUSTOMIZE="${KUSTOMIZE:-kustomize}"
+
+# Override the image name when this is invoked from openshift ci
+if [ -n "${OPENSHIFT_BUILD_NAMESPACE}" ]; then
+  FULL_REGISTRY_IMAGE="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:performance-addon-operators-registry"
+fi
+
+echo "Deploying using image $FULL_REGISTRY_IMAGE."
+
+# Deploy features
+success=0
+iterations=0
+sleep_time=10
+max_iterations=30 # results in 5 minute timeout
+feature_dir=cluster-setup/ci-cluster/performance/
+
+until [[ $success -eq 1 ]] || [[ $iterations -eq $max_iterations ]]
+do
+
+  echo "[INFO] Deploying performance operator and profile."
+  set +e
+  if ! ${KUSTOMIZE} build $feature_dir | envsubst | ${OC_TOOL} apply -f -
+  then
+    set -e
```

Comment:
should be removed or here or on line 39

Reply:
we don't reach line 39 in this if branch, so I need to reset it twice